libkbfs/tlf_journal.go

```diff
@@ -436,6 +476,7 @@ func (j *tlfJournal) doBackgroundWorkLoop(
 				j.log.CDebugf(ctx,
 					"Got pause signal for %s", j.tlfID)
 				bws = TLFJournalBackgroundWorkPaused
+				j.pause(journalPausedFromSignal)
```

Comment:
this and the call below would go away, after implementing my comment above re. combining `pause` and `pauseBackgroundWork`